components/pipeline-service/base/certificates/tekton-results/tekton-results-db-ssl.yaml

```diff
@@ -0,0 +1,132 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: rds-root-crt
+data:
+# Public certificate for RDS instances are pubically available to download here: 
+# https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.SSL.html#UsingWithRDS.SSL.RegionCertificates
+  tekton-results-db-ca.pem: |-
+    -----BEGIN CERTIFICATE-----
```

Comment:
don't certs like this need to be injected into vault and accessed by external secrets?